### PR TITLE
[SCB-2476] Split checkstyle to a independent task to speed up ci

### PR DESCRIPTION
**Pull request by Shoothzj**
## changes
- split checkstyle task
- findbugs is an abandoned project, and we now have spotbugs. so remove the `findbugs`

### CODE REVIEW

File: .github/workflows/checkstyle.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up jdk
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: spotbugs
+        run: mvn -B -Pit install -DskipTests -Dspotbugs.skip=true checkstyle:checkstyle
```
Comment (liubao68) [inline .github/workflows/checkstyle.yml:37]: Is `install` requried?

Comment (Shoothzj) [reply .github/workflows/checkstyle.yml:37]: @liubao68 Yes, only checkstyle can't resolve dependencies, can't use in multi module project

Comment (liubao68) [reply .github/workflows/checkstyle.yml:37]: `spotbugs` vs `checkstyle` seems diffirenct plugin

Comment (Shoothzj) [reply .github/workflows/checkstyle.yml:37]: @liubao68 miss change the name, fixed

**Review by liubao68 — APPROVED** [review]

**Pull request merged**